src/store/plugins/names.js

```diff
@@ -126,10 +126,10 @@ export default (store) => {
         { rootGetters: { activeNetwork, accounts }, commit },
       ) {
         await Promise.all(accounts.map(async ({ address }) => {
-          const { preferredChainName: defaultNameBackend } = await fetchJson(
+          const response = await fetchJson(
             `${activeNetwork.backendUrl}/profile/${address}`,
-          ).catch(() => null);
-          commit('setDefault', { address, name: defaultNameBackend });
+          ).catch(() => {});
+          commit('setDefault', { address, name: response?.preferredChainName });
```

Comment:
`?` won't' be needed if `fetchJson` would throw an error in case not 200 response

Reply:
It is in the scope of PR that will introduce fetchJson throwing issues.